### PR TITLE
add slack channel and format totems info output

### PR DESCRIPTION
**Pull request by 0xmaayan**
Adding slack channel and formatting to `totems info` / `totems info <totem>`

image

image

### CODE REVIEW

File: lib/lita/handlers/totems.rb [hunk 1]
```diff
@@ -7,6 +7,24 @@ module Handlers
     class Totems < Handler
 
       @@DemoEnvironments = %w(cyan jade noir opal plum teal vert)
+      @@EnvironmentToSlackChannelMap = {
```
Comment (0xmaayan) [inline lib/lita/handlers/totems.rb:10]: not completely sure if this is the right way to hold a map. my experience is with `JS` where I would probably create a `js/json` file that holds this map and use it from there, couldn't find the "right" way to do it with Ruby

Comment (0xmaayan) [reply lib/lita/handlers/totems.rb:10]: also, this list is not complete and we are probably missing some relations (will add it later)

**Review by MurasakiArtemis — APPROVED** [review]
LGTM (I saw it working on Changebot Demo yesterday)

File: lib/lita/handlers/totems.rb (hunk 1 above)
Comment (MurasakiArtemis) [reply lib/lita/handlers/totems.rb:10]: Don't worry too much, probably later we'll add some linting, but as far as I know, this is the right way to hold a map, if you wanted to have it in another file you'll have to import the variable 🤔

**Pull request merged**